src/components/ExpensesList.js

```diff
@@ -0,0 +1,27 @@
+import Expense from "./Expense";
+
+const ExpensesList = ({ expenses, removeExpense }) => {
+  const renderExpenses = () => {
```

Comment:
@MatthewBerglund: one of the good things of functional components is that you do not really need `render()` methods anymore. You can `map` through your `expenses` in line 21:

`{expenses?.length > 0 ? expenses.map(... => <Expense ..... />) : <p>You have ....</p>}`

Render() methods are only used with class components. With functional components JSX makes most of it for you.

Reply:
This was just an attempt on my part to avoid a multiline ternary and potential readability issues. I'm not sure if I understand how using a function in this way is reserved for class components only. Could you explain?

Reply:
Sure. Probably I was a bit too direct and itchy in regards to the function `render()` because my brain connects it immediately to class components. I thought you wanted to use it like we normally do in class components but inside a functional component. Now I better understand what you wanted to do.

This is again a matter of personal taste in terms of readability, in this case I do not think the ternary makes it so complicated but other options could be for example using two `&&` operators:
`{expenses?.length === 0 && <p>You have ....</p>}`
`{expenses?.length > 0 && expenses.map(... => <Expense ..... />)}`

Reply:
:(